tools/util.wdl

```diff
@@ -370,47 +370,55 @@ task unpack_tarball {
     }
 }
 
-task make_coverage_regions_beds {
-    # TODO make this customizable
+task make_coverage_regions_bed {
```

Comment:
This is certainly out-of-scope, but I was recently extracting `protein_coding` genes from an annotation file. Should we consider generalizing this in the future to extract any given set of features from an annotation GTF file?

Reply:
So this PR "generalized" the extraction to enable pulling out any specific values from the `feature_type` column. If I understand the request, you want to control which column the match is performed on as well as the value matched on? That _should_ be straightforward. We'd want to restrict to columns that make sense to compare on (e.g. _not_ the `start`/`end` columns), but that should be pretty easy.

Reply:
Right, I was thinking of a more general `extract_features` type task. Much of that logic is here though, so I wouldn't necessarily want to duplicate it. However as written, since it converts to BED, you lose the other annotations. So if you extract `gene` features, you can't subsequently refine to only `protein_coding` genes.

Reply:
We'd have to parse the annotation field to find the `gene_type` field (for GENCODE). That may not be portable though, so maybe it wouldn't be worth it.